setup/README.adoc

```diff
@@ -0,0 +1,85 @@
+= Dev Sandbox Setup
+
+This document describes how to setup a Dev Sandbox environment that is similar to production and can be used as part of an onboarding process for new operators.
+
+=== Prereqs
+
+1. Provision an OCP cluster with sufficient resources.
+
+For example, if using AWS:
+
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform:
+    aws:
+      type: "m5.4xlarge"
+  replicas: 3
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      type: "m5.2xlarge"
+  replicas: 3
+metadata:
+  name: sandbox-test
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  aws:
+    region: us-east-1
+publish: External
+----
+
+2. Install your operator(s)
+
+3. Create a user-workloads.yaml file. This file should contain resources that the setup tool will automatically create in user namespaces. Use the sample user-workloads.yaml 🚧 coming soon 🚧  as a base and add custom resources that your operator uses.
+
+4. Complete the following steps:
+* Make sure you have set the `QUAY_NAMESPACE` variable: `export QUAY_NAMESPACE=<quay-username>`
+* Log in to the quay.io using `docker login quay.io`
+* Make sure that the visibility of all repositories `host-operator`, `member-operator` and `registration-service` in quay is set to `public` (https://quay.io/repository/<your-username>/host-operator?tab=settings https://quay.io/repository/<your-username>/member-operator?tab=settings https://quay.io/repository/<your-username>/registration-service?tab=settings)
+* Log in to the target OpenShift 4.2+ cluster with cluster admin privileges using `oc login`
+
+=== Dev Sandbox Setup
+
+🚧 This section is still a work in progress 🚧
+
+1. Clone this repository
+`git clone git@github.com:codeready-toolchain/toolchain-e2e.git`
+2. Run the following to install the Dev Sandbox operators
+`make deploy-e2e HOST_NS=toolchain-host-operator MEMBER_NS=toolchain-member-operator MEMBER_NS_2=toolchain-member2-operator`
```

Comment:
The default ones when using the `deploy-e2e` target use the naming pattern `${QUAY_NAMESPACE}-host-${DATE_SUFFIX}`. eg. `rsenthil-host-11085720` So it looks a bit random because of the DATE_SUFFIX, I figured it's better to have something more constant.

I changed the instructions to use `dev-deploy-e2e` target instead and then the default naming pattern is `${QUAY_NAMESPACE}-host-operator` eg. `rsenthil-host-operator` which is a little better.